api/src/pcapi/sandboxes/scripts/creators/test_cases/__init__.py

```diff
@@ -369,3 +373,124 @@ def create_multiauthors_books(venues: list[offerers_models.Venue]) -> None:
     author = "collectif"
     for _ in range(3):
         create_offer_with_ean(Fake.ean13(), random.choice(venues), author=author)
+
+
+def create_offers_with_more_extra_data() -> None:
+    venue_data = random.choice(venues_mock.venues)
+    venue = offerers_factories.VenueFactory(
+        name="extra_data " + str(venue_data["name"]),
+        venueTypeCode=offerers_models.VenueTypeCode.BOOKSTORE,
+        isPermanent=True,
+        latitude=venue_data["latitude"],
+        longitude=venue_data["longitude"],
+        address=venue_data["address"],
+        postalCode=venue_data["postalCode"],
+        city=venue_data["city"],
+        departementCode=venue_data["departementCode"],
+    )
+    for _ in range(2):
+        create_offers_with_extradata(
+            venue=venue,
+            extra_data={
+                "author": Fake.name(),
+                "ean": Fake.ean13(),
+                "gtl_id": random.choice(list(GTLS.keys())),
+                "editeur": Fake.name(),
+            },
+            subcategory=subcategories_v2.LIVRE_PAPIER,
+            should_create_product=True,
+        )
+        create_offers_with_extradata(
+            venue=venue,
+            extra_data={
+                "cast": [Fake.name() for _ in range(random.randint(1, 10))],
+                "releaseDate": Fake.date(),
+                "genres": [MOVIE_TYPES[random.choice(list(MOVIE_TYPES))] for _ in range(random.randint(1, 4))],
+                "stageDirector": Fake.name(),
+            },
+            subcategory=subcategories_v2.SEANCE_CINE,
+        )
+        music_type = random.choice(music_types)
+        for subcategory in [
+            subcategories_v2.SUPPORT_PHYSIQUE_MUSIQUE_CD,
+            subcategories_v2.FESTIVAL_MUSIQUE,
+            subcategories_v2.CAPTATION_MUSIQUE,
+            subcategories_v2.EVENEMENT_MUSIQUE,
+            subcategories_v2.LIVESTREAM_MUSIQUE,
+            subcategories_v2.TELECHARGEMENT_MUSIQUE,
+            subcategories_v2.SUPPORT_PHYSIQUE_MUSIQUE_VINYLE,
+            subcategories_v2.ACHAT_INSTRUMENT,
+            subcategories_v2.LOCATION_INSTRUMENT,
+            subcategories_v2.CONCERT,
+            subcategories_v2.EVENEMENT_MUSIQUE,
+            subcategories_v2.FESTIVAL_MUSIQUE,
+        ]:
```

Comment:
<!-- Thoughts represent an idea that popped up from reviewing. These comments are non-blocking by nature, but they are extremely valuable and can lead to more focused initiatives and mentoring opportunities.  -->
**💭 thought:** ‏On peut éviter la liste ici en récupérant les sous-catégories qui ont `MUSIC_TYPE` dans leur `conditional_fields`
A extraire dans une fonction si on fait ça.
L'avantage c'est que la sandbox suivra si on change les sous-cat